src/bit-components.js

```diff
@@ -178,7 +178,28 @@ MediaPDF.map = new Map();
 export const MediaVideo = defineComponent({
   autoPlay: Types.ui8
 });
-export const AnimationMixer = defineComponent();
+export const MixerAnimatableInitialize = defineComponent({});
+export const MixerAnimatable = defineComponent({});
+/**
+ * @type {Map<EntityId, AnimationMixer}>}
+ */
+export const MixerAnimatableData = new Map();
```

Comment:
Would prefer we start putting component properties on this on the component and just using type annotations, like https://github.com/mozilla/hubs/blob/behavior-graphs-spike/src/bit-components.js#L185-L190